config/localTemplate.go

```diff
@@ -454,28 +454,16 @@ type Local struct {
 	// VerifiedTranscationsCacheSize defines the number of transactions that the verified transactions cache would hold before cycling the cache storage in a round-robin fashion.
 	VerifiedTranscationsCacheSize int `version[14]:"30000" version[23]:"150000"`
 
-	// EnableCatchupFromArchiveServers controls which peers the catchup service would use in order to catchup.
-	// When enabled, the catchup service would use the archive servers before falling back to the relays.
-	// On networks that don't have archive servers, this becomes a no-op, as the catchup service would have no
-	// archive server to pick from, and therefore automatically selects one of the relay nodes.
-	EnableCatchupFromArchiveServers bool `version[15]:"false"`
```

Comment:
Could you check json deserializes OK when these fields present? I think the decoder is configured to skip unknown fields but it is better to double check

Reply:
Checked, the test runs older config json versions that have these fields present (as well as the isIndexerActive that was removed last year) - see https://github.com/algorand/go-algorand/blob/master/config/config_test.go#L239 for the test case.